governance/cla/index.html

```diff
@@ -0,0 +1,3 @@
+---
+redirect_to: "https://libero.pub/governance/cla/cla.pdf"
```

Comment:
Wonder if a relative URI works?

Reply:
Docs suggest it will

>If you are hosting your Jekyll site on GitHub Pages, and site.url is not set, the prefix is set to the pages domain name i.e. http://example.github.io/project or a custom CNAME.

Have updated in https://github.com/libero/libero.github.io/pull/1/commits/0b341c20aeaaa3e707616a5bcda1446890be7134